2017/AUG/AUG17/GCAC/GCAC.cpp

```diff
@@ -0,0 +1,47 @@
+//Made by vanguard7
+#include<bits/stdc++.h>
+using namespace std;
+
+int main() {
+	ios::sync_with_stdio(false);
+	cin.tie(NULL);
+	int t;
+	cin>>t;
+	while(t--) {
+		int n, m, i, j;
+		set<int>s;
+		cin>>n>>m;
+		char x[m+1];
+		long long int minSalary[n], offeredSalary[m], maxJobOffered[m], cost=0, jobs=0, companies = m;
+		for(i=0; i<n; i++)
```

Comment:
Try to use `++i` instead of `i++`, as this repo is concerned towards competitve programming. So the solution should be fast and efficient in terms of solution.

Follow this [link](https://stackoverflow.com/questions/24901/is-there-a-performance-difference-between-i-and-i-in-c)